lib/xander.ex

```diff
@@ -2,6 +2,6 @@ defmodule Xander do
   @moduledoc false
 
   def get_current_era do
-    Xander.ClientStatem.query(:get_current_era)
+    Xander.Query.run(:get_current_era)
```

Comment:
this namespace looks good / is descriptive when in the Supervisor children keyword list :+1: